commit 22a010eb786101e93b0e9dcd646d06c7fb419782 (refs/changes/05/1905/1)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-05-04T20:20:12+00:00 (2 years, 5 months ago)
    
    NAPI-413 Update sdc-portolan-moray dependency to pull in NAPI-401 and NAPI-412

diff --git a/package.json b/package.json
index 4a9688d..05657fc 100644
--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "lomstream": "1.1.0",
     "moray": "2.0.1",
     "node-uuid": "1.4.7",
-    "portolan-moray": "git+https://github.com/joyent/sdc-portolan-moray.git#5f266f3",
+    "portolan-moray": "git+https://github.com/joyent/sdc-portolan-moray.git#7e2c4ca",
     "restify": "4.1.1",
     "restify-warden": "0.1.1",
     "sdc-clients": "9.2.0",
